SH4RE/SH4RE/ViewListing/ViewListingView.swift

```diff
@@ -34,62 +34,66 @@ struct ViewListingView: View {
         
         ZStack {
             Color("BackgroundGrey").ignoresSafeArea()
-            VStack(alignment: .leading) {
-                
-                GeometryReader { geometry in
-                    ImageCarouselView(numberOfImages: self.numberOfImages) {
-                        ForEach([self.image, UIImage(named: "ProfilePhotoPlaceholder")!, UIImage(named: "ProfilePhotoPlaceholder")!], id:\.self) { image in
-                            Image(uiImage: image)
-                                .resizable()
-                                .scaledToFit()
-                                .frame(width: geometry.size.width, height: 250)
-                                .aspectRatio(contentMode: .fill)
+            ScrollView {
+                VStack(alignment: .leading) {
+                    
+                    GeometryReader { geometry in
+                        ImageCarouselView(numberOfImages: self.numberOfImages) {
+                            ForEach([self.image, UIImage(named: "ProfilePhotoPlaceholder")!, UIImage(named: "ProfilePhotoPlaceholder")!], id:\.self) { image in
+                                Image(uiImage: image)
+                                    .resizable()
+                                    .scaledToFit()
+                                    .frame(width: geometry.size.width, height: 250)
+                                    .aspectRatio(contentMode: .fill)
+                            }
                         }
                     }
-                }
-                
-                Text(self.title)
-                    .font(.title)
-                    .fontWeight(.bold)
-                    .padding([.horizontal])
-                
-                HStack {
-                    StarsView(numberOfStars: self.numberOfStars)
                     
-                    Text("(\(self.numberOfReviews) reviews)")
-                        .font(.caption)
-                        .foregroundColor(Color("TextGrey"))
-                }
-                .padding([.horizontal])
-                
-                Text(self.description)
-                    .font(.body)
-                    .padding([.horizontal])
-                    .padding([.top], 10)
-                
-                Button(action: {
-                    print("Check Availability button clicked")
-                }) {
+                    Color.white.frame(width: screenSize.width, height: 275).opacity(0)
+                    
+                    Text(self.title)
```

Comment:
is this wasifs code? why all the self?

Reply:
[Heres what they say about using self](https://dmitripavlutin.com/how-to-use-correctly-self-keyword-in-swift/#:~:text=The%20answer%20is%20that%20Swift%20permits%20and%20even%20encourages%20you%20to%20omit%20self%20keyword%20when%20it%20can%20be%20done.%20The%20challenge%20is%20to%20determine%20the%20scenarios%20when%20self%20is%20obligatory%20and%20when%20optional)

Reply:
i changed!